cookiecutter/cli.py

```diff
@@ -48,6 +49,33 @@ def validate_extra_context(ctx, param, value):
     return dict(s.split('=', 1) for s in value) or None
 
 
+def list_installed_templates(default_config, passed_config_file):
+    """
+    Lists installed (locally cloned) templates. Use cookiecutter installed
+    """
+
+    config = get_user_config(passed_config_file, default_config)
+    cookiecutter_folder = config.get('cookiecutters_dir')
+
+    if not os.path.exists(cookiecutter_folder):
+        click.echo('Error: Cannot list installed templates. ' +
+                   'Folder does not exist: {}'.format(cookiecutter_folder))
+        sys.exit(-27)
```

Comment:
I simply wanted a different code than other errors so things scripting cookiecutter can use that as a signal for why / how something failed. If everywhere you return -1 how do you know what went wrong?